<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeytrackSerial</class>
 <widget class="QWidget" name="KeytrackSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>561</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Received</string>
   </property>
   <widget class="QTextEdit" name="receivedBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>561</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Sent</string>
   </property>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>471</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sentBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="portList">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port Name</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Port Settings</string>
   </property>
   <widget class="QComboBox" name="stopBits">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="parity">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
